birdhouse/components/thredds/default.env

```diff
@@ -17,7 +17,22 @@ export THREDDS_SERVICE_DATA_LOCATION_NAME='Birdhouse'
 export THREDDS_DATASET_URL_PATH='datasets'
 export THREDDS_SERVICE_DATA_URL_PATH='birdhouse'
 
+export THREDDS_SERVICE_DATA_EXTRA_FILE_FILTERS=''
 
+export THREDDS_DATASET_DATASETSCAN_BODY='
+      <metadata inherited="true">
+        <serviceName>all</serviceName>
+      </metadata>
+
+      <filter>
+        <include wildcard="*.nc" />
+        <include wildcard="*.ncml" />
+        <include wildcard="*.txt" />
+        <include wildcard="*.md" />
+        <include wildcard="*.rst" />
+        <include wildcard="*.csv" />
+      </filter>
```

Comment:
`THREDDS_SERVICE_DATA_EXTRA_FILE_FILTERS` should probably be embedded by default within `THREDDS_DATASET_DATASETSCAN_BODY`? 

I think the original intention was to always update the 2 datasets with similar combinations of extensions. If not desired, the user can still override everything, but it does need for them to override everything if they only want to add 1 other extension via `THREDDS_SERVICE_DATA_EXTRA_FILE_FILTERS`.